src/commands/kv/delete_bulk.rs

```diff
@@ -0,0 +1,99 @@
+extern crate base64;
+
+use cloudflare::framework::apiclient::ApiClient;
+use percent_encoding::{percent_encode, PATH_SEGMENT_ENCODE_SET};
+use walkdir::WalkDir;
+
+use std::ffi::OsString;
+use std::fs;
+use std::fs::metadata;
+use std::path::Path;
+
+use cloudflare::endpoints::workerskv::delete_bulk::DeleteBulk;
+use failure::bail;
+
+use crate::terminal::message;
+
+pub fn delete_bulk(namespace_id: &str, filename: &Path) -> Result<(), failure::Error> {
+    let client = super::api_client()?;
+    let account_id = super::account_id()?;
+
+    // If the provided argument for delete_bulk is a json file, parse it
+    // and delete its listed keys. If the argument is a directory, delete key-value
+    // pairs where keys are the relative pathnames of files in the directory.
+    let mut data;
+    let keys: Result<Vec<String>, failure::Error> = match metadata(filename) {
+        Ok(ref file_type) if file_type.is_file() => {
+            data = fs::read_to_string(filename)?;
+            Ok(serde_json::from_str(&data)?)
+        }
+        Ok(ref file_type) if file_type.is_dir() => parse_directory(filename),
+        Ok(_file_type) => {
+            // any other file types (namely, symlinks)
+            bail!(
+                "{} should be a file or directory, but is a symlink",
+                filename.display()
+            )
+        }
+        Err(e) => bail!(e),
+    };
+
+    let response = client.request(&DeleteBulk {
+        account_identifier: &account_id,
+        namespace_identifier: namespace_id,
+        bulk_keys: keys?,
```

Comment:
nit: i think having `?` at variable declaration rather than in a struct constructor feels... more right? So at the end of ln39? idk, @steveklabnik or @ashleygwilliams may advise.